internal/workos/http.go

```diff
@@ -25,23 +27,33 @@ func TryGetHTTPError(r *http.Response) error {
 	} else {
 		msg = string(body)
 	}
+	if e := gjson.GetBytes(body, "error").Str; e != "" {
+		error = e
+	}
+	if d := gjson.GetBytes(body, "error_description").Str; d != "" {
+		description = d
+	}
 
 	return HTTPError{
-		Code:      r.StatusCode,
-		Status:    r.Status,
-		RequestID: r.Header.Get("X-Request-ID"),
-		Message:   msg,
+		Code:           r.StatusCode,
+		Status:         r.Status,
+		RequestID:      r.Header.Get("X-Request-ID"),
+		Message:        msg,
+		Err:            error,
+		ErrDescription: description,
 	}
 }
 
 // HTTPError represents an http error.
 type HTTPError struct {
-	Code      int
-	Status    string
-	RequestID string
-	Message   string
+	Code           int
+	Status         string
+	RequestID      string
+	Message        string
+	Err            string
+	ErrDescription string
 }
 
 func (e HTTPError) Error() string {
-	return fmt.Sprintf("%s: request id %q: %s", e.Status, e.RequestID, e.Message)
+	return fmt.Sprintf("%s: request id %q: %s %s %s", e.Status, e.RequestID, e.Message, e.Err, e.ErrDescription)
```

Comment:
Some of these properties are going to be mutually-exclusive (for example, we won't have a `message` alongside an `error` and `error_description.`

We should probably adjust the string representation based on which properties are present.